### PR TITLE
Internal: Conversion Center - Address Feedback and Refactor class for Pro

### PR DESCRIPTION
**Pull request by nicoladj77**
## PR Checklist
- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Address Feedback related to Link In bio
* Refactor rendering classes to allow better extendability

## Description
An explanation of what is done in this PR

* Address Feedback related to Link In bio
* Refactor rendering classes to allow better extendability

## Test instructions
This PR can be tested by following these steps:

* Add the Link In bio Widget and verify that things have been addressed

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/conversion-center/widgets/link-in-bio.php [hunk 1]
```diff
@@ -849,7 +917,8 @@ private function add_style_tab(): void {
 				'label'     => esc_html__( 'Text Color', 'elementor' ),
 				'type'      => Controls_Manager::COLOR,
 				'selectors' => [
-					'{{WRAPPER}} .e-link-in-bio' => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
+					'{{WRAPPER}} .e-link-in-bio'                   => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
+					'{{WRAPPER}} .e-link-in-bio__cta.is-type-link' => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
```
Comment (louiswol94) [inline modules/conversion-center/widgets/link-in-bio.php:921]:
```suggestion
					'{{WRAPPER}}'  => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
```
Does it work if you just target the wrapper? When using CSS variables in general, I think we just need the `{{WRAPPER}}`

File: modules/conversion-center/widgets/link-in-bio.php [hunk 2]
```diff
@@ -1189,7 +1258,7 @@ private function add_advanced_tab() {
 
 		foreach ( $active_breakpoints as $breakpoint_key => $breakpoint ) {
 			$available_devices[ $breakpoint_key ] = $breakpoint->get_label();
-			$default_devices[] = $breakpoint_key;
+			$default_devices[]                    = $breakpoint_key;
```
Comment (louiswol94) [inline modules/conversion-center/widgets/link-in-bio.php:1261]:
```suggestion
			$default_devices[] = $breakpoint_key;
```
Please undo?

File: modules/conversion-center/widgets/link-in-bio.php [hunk 3]
```diff
@@ -1221,33 +1290,33 @@ private function add_advanced_tab() {
...
 			'advanced_custom_css_id',
 			[
-				'label' => esc_html__( 'CSS ID', 'elementor' ),
-				'type' => Controls_Manager::TEXT,
-				'default' => '',
-				'ai' => [
+				'label'          => esc_html__( 'CSS ID', 'elementor' ),
+				'type'           => Controls_Manager::TEXT,
+				'default'        => '',
+				'ai'             => [
```
Comment (louiswol94) [inline modules/conversion-center/widgets/link-in-bio.php:1296]: @nicoladj77 Can we maybe in a separate PR remove all these spaces - it's more consistent with the rest of the code base, and I see now it creates bigger diffs in PRs.

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: modules/conversion-center/widgets/link-in-bio.php (hunk 1 above)
Comment (nicoladj77) [reply modules/conversion-center/widgets/link-in-bio.php:921]: @willhowat told me that this is not working if he only targets the wrapper, correct @willhowat ?

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**